src/PostCampaign.php

```diff
@@ -33,7 +33,8 @@ class PostCampaign {
 		'p4_title',
 		'p4_subtitle',
 		'p4_description',
-		'background_image',
+		'background_image_id',
+		'background_image_url',
```

Comment:
The old meta fields saved in DB was `background_image_id` & `background_image`, so if you edit old page header fields the image field will looks empty.
In my opinion its good to keep the same meta-field name or we need to add backward compatibility for the name change 🤷‍♂️

Reply:
Good point, updated!

Reply:
PR looks good to me 👍